tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1461,6 +1461,12 @@ voq/test_fabric_reach.py:
     conditions:
       - "('t2' not in topo_name) or (asic_subtype not in ['broadcom-dnx']) or ('arista_7800' not in platform)"
 
+voq/test_voq_fabric_isolation.py:
+  skip:
+    reason: "Skip test_voq_fabric_isolation on unsupported testbed."
+    conditions:
+      - "('t2' not in topo_name) or (asic_subtype not in ['broadcom-dnx']) or ('arista_7800' not in platform)"
```

Comment:
please remove the arista-7800 check, so that the test can be run on all DNX platforms

Reply:
> please remove the arista-7800 check, so that the test can be run on all DNX platforms

done